docs/build/interoperate-qiskit-qasm3.mdx

```diff
@@ -31,7 +31,7 @@ You must install the package `qiskit_qasm3_import ` to use this function.  For e
 pip install qiskit-qasm3-import
 ```
 
-In this example, we define a quantum program using OpenQASM 3, and use `loads()` to directly convert it into a QuantumCircuit:
+In this example, we define a quantum program using OpenQASM 3, and use `qasm3.loads()` to directly convert it into a QuantumCircuit:
```

Comment:
same here, throughout these pages we should decide if we're going to use just the function name or include the `qasm3` bit. we should keep i consistent everywhere, so here I'd recommend dropping the `qasm3` and just saying `loads()`

Reply:
done in https://github.com/Qiskit/documentation/pull/239/commits/b05663072a19831a7587d7822abe40cbacb175a0